src/main/java/dev/morling/kccli/command/DescribeConnectorCommand.java

```diff
@@ -85,17 +93,37 @@ public void run() {
             config.add(new Tuple("  " + configEntry.getKey(), configEntry.getValue()));
         }
 
-        printTuples(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Config" + ANSI_RESET, "")));
-        printTuples(config);
+        Tuple.print(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Config" + ANSI_RESET, "")));
+        Tuple.print(config);
 
-        printTuples(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Topics" + ANSI_RESET, "")));
+        Tuple.print(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Topics" + ANSI_RESET, "")));
 
         List<Tuple> topics = new ArrayList<>();
 
         for (String topic : connectorTopics.entrySet().iterator().next().getValue().topics) {
             topics.add(new Tuple("", "  " + topic));
         }
-        printTuples(topics);
+        Tuple.print(topics);
+
+        if (includeTasksConfig) {
+            Version currentVersion = new Version(kafkaConnectApi.getWorkerInfo().version);
+            Tuple.print(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Tasks Config" + ANSI_RESET, "")));
+            if (currentVersion.greaterOrEquals(requiredVersionForTasksConfig)) {
+                Map<String, Map<String, String>> tasksConfigs = kafkaConnectApi.getConnectorTasksConfig(name);
+
+                List<Tuple> tuples = new ArrayList<>();
+                for (Entry<String, Map<String, String>> task : tasksConfigs.entrySet()) {
+                    tuples.add(new Tuple("  Task", task.getKey()));
+                    for (Entry<String, String> taskConfig : task.getValue().entrySet()) {
+                        tuples.add(new Tuple("    " + taskConfig.getKey(), taskConfig.getValue()));
+                    }
+                }
+                Tuple.print(tuples);
+            }
+            else {
+                System.out.println("  --tasks-config requires at least Kafka Connect 2.8. Current version: " + currentVersion);
```

Comment:
So do you think we actually should show the other output in this situation; or should we rather handle this as sort of an invocation failure more explicitly and _only_ show this message when passing the parameter with KC < 2.8?